guidelines/pride-affinity-proteomics.md

```
@@ -7,185 +7,196 @@
-        policies](#pride-archive-guidelines-and-policies)
+  policies](#pride-archive-guidelines-and-policies)
 - [Supported technologies](#supported-technologies)
 - [Dataset structure and file
-        requirements](#dataset-structure-and-file-requirements)
-   - [Accession](#accession)
-   -   [Dataset metadata](#dataset-metadata)
-   -   [File requirements](#file-requirements)
+  requirements](#dataset-structure-and-file-requirements)
+    - [Accession](#accession)
+    -   [Dataset metadata](#dataset-metadata)
+    -   [Olink and SomaScan file requirements](#olink-and-somaScan-file-requirements)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)